tests/lax_numpy_indexing_test.py

```diff
@@ -871,6 +871,18 @@ def testTrivialGatherIsntGenerated(self):
     jaxpr = jax.make_jaxpr(lambda x: x[:4])(np.arange(4))
     self.assertEqual(len(jaxpr.jaxpr.eqns), 0)
 
+  @parameterized.named_parameters(
+      {"testcase_name": f"_{idx_type_name}_{idx}", "idx": idx, "idx_type": idx_type}
+      for idx in (-3, 0, 5)
+      for idx_type_name, idx_type in (
+        ("int", int), ("np.array", np.array), ("jnp.array", jnp.array)))
```

Comment:
Let me know if there is a better way to get the `idx_type` name automatically for the test name.